agents-api/agents_api/clients/litellm.py

```diff
@@ -53,7 +53,7 @@ async def acompletion(
         model = f"openai/{model}"  # This is needed for litellm
 
     supported_params = get_supported_openai_params(model)
-    settings = {k: v for k, v in kwargs.items() if k in supported_params}
+    settings = {k: v for k, v in kwargs.items() if k in supported_params or k == "user"}
```

Comment:
this is lousy, instead I'd do:

```python
    supported_params: list[str] = get_supported_openai_params(model) or []  # Supported params returns Optional[list[str]]
    supported_params += ["user"]
    settings = {k: v for k, v in kwargs.items() if k in supported_params}
 ```

Reply:
@ellipsis-dev please apply this change

Reply:
I would use set instead of list (minor), otherwise looks good